test/unit/entry.spec.ts

```diff
@@ -65,6 +70,24 @@ describe('createEntry()', () => {
       })
     })
 
+    describe('entry methods', () => {
+      describe('calls task API methods', () => {
+        ;['getTasks', 'getTask', 'createTask', 'updateTask', 'deleteTask'].forEach((methodName) => {
+          const args = ['foo', 42, {}]
+          describeChannelCallingMethod({
+            creator: createEntry,
+            methodName,
+            channelMethod: 'callEntryMethod',
+            args: args,
+            expectedCallArgs: [methodName, args],
+            makeCreatorArgs: () => {
+              return [{}, [{}], () => ({})]
+            },
```

Comment:
`makeCreatorArgs` is confusing to me.
How about

```javascript
creator: (channel) => createEntry(channel, {}, [{}], () => ({}))
```

and removing that again. It feels like it makes it unnecessarily complex